packages/compiler/test/checker/effective-type.test.ts

```diff
@@ -198,42 +229,7 @@ describe("compiler: effective type", () => {
     strictEqual(propType?.kind, "Model" as const);
 
     const effective = testHost.program.checker.getEffectiveModelType(propType, removeFilter);
-    strictEqual(effective, Derived);
-  });
-
-  it("does not depend on property order", async () => {
```

Comment:
This test was weird (reversing property list) and the new algorithm isn't sensitive to property order so I just deleted it. It passes still, but it's just not interesting to keep around.